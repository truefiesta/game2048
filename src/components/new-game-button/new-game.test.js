import React from "react";
import renderer from "react-test-renderer";
import NewGameButton from "./new-game-button";

describe(`NewGameButton snapshot`, () => {
    it(`renders`, () => {
        const newGameButton = renderer.create(
            <NewGameButton/>
        )
        .toJSON();

        expect(newGameButton).toMatchSnapshot();
    });
});
