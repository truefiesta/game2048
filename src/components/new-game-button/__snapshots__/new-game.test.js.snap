// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewGameButton snapshot renders 1`] = `
<button
  className="new-game-button"
>
  New Game
</button>
`;
